test/test_distributions.py

```diff
@@ -203,6 +203,43 @@ def test_normal_sample(self):
                                         scipy.stats.norm(loc=mean, scale=std),
                                         'Normal(mean={}, std={})'.format(mean, std))
 
+    def test_exponential(self):
+        rate = Variable(torch.randn(5, 5).abs(), requires_grad=True)
+        rate_1d = Variable(torch.randn(1).abs(), requires_grad=True)
+        self.assertEqual(Exponential(rate).sample().size(), (5, 5))
+        self.assertEqual(Exponential(rate).sample_n(7).size(), (7, 5, 5))
```

Comment:
Adam suggested we use `.sample((n,))` rather than `.sample_n(n)` in all new tests.